vsintegration/src/FSharp.Editor/Navigation/GoToDefinitionService.fs

```diff
@@ -30,74 +45,99 @@ type internal FSharpNavigableItem(document: Document, textSpan: TextSpan) =
         member this.DisplayTaggedParts = ImmutableArray<TaggedText>.Empty
         member this.ChildItems = ImmutableArray<INavigableItem>.Empty
 
+
 [<Shared>]
 [<ExportLanguageService(typeof<IGoToDefinitionService>, FSharpCommonConstants.FSharpLanguageName)>]
-type internal FSharpGoToDefinitionService 
-    [<ImportingConstructor>]
-    (
-        checkerProvider: FSharpCheckerProvider,
-        projectInfoManager: ProjectInfoManager,
-        [<ImportMany>]presenters: IEnumerable<INavigableItemsPresenter>
+type internal FSharpGoToDefinitionService [<ImportingConstructor>]
+    (   checkerProvider: FSharpCheckerProvider
+    ,   projectInfoManager: ProjectInfoManager
+    ,   signatureMetadataService : NavigateToSignatureMetadataService
+    ,   [<ImportMany>]_presenters: IEnumerable<INavigableItemsPresenter>
```

Comment:
We should keep any style which doesn't look unkept and look like some work was put to layout the code for better reading.

Trying to harmonize the codebase at large to a single style doesn't seem bring as much as just making sure code has had efforts to make readable.

I prefer this (by far) versus all on single or two lines.